### PR TITLE
override config params

### PR DESCRIPTION
**Pull request by Ilya-Krylov**

### CODE REVIEW

**Comment by Ilya-Krylov** [comment]
run tests

File: tools/test.py [hunk 1]
```diff
@@ -257,6 +260,23 @@ def main():
...
+            stack = []
+            for ki in k:
+                stack.append(cfg_el)
+                cfg_el = cfg_el[ki]
+            cfg_el = v
+            for ki in k[::-1]:
+                a = stack.pop()
+                a[ki] = cfg_el
+                cfg_el = a
+            cfg = cfg_el
```
Comment (druzhkov-paul) [inline tools/test.py:278]: I don't mind this change now. Though would rather replace it with [mmcv's implementation](https://github.com/open-mmlab/mmcv/blob/master/mmcv/utils/config.py#L342-L403) as soon as upstream is merged.

File: tools/test.py [hunk 2]
```diff
@@ -237,6 +237,9 @@ def parse_args():
                         help='Enables multi-scale testing: h1 w1 h2 w2 ...')
     parser.add_argument('--tta_flip', action='store_true',
                         help='Enables flip as one of test-time augmentations.')
+    parser.add_argument('--update_config', nargs='+',
+                        help='It can be used to override configuration file parameters. '
+                             'Usage: --update_config total_epochs=75, data.val.ann_file=my_annotation.json')
```
Comment (druzhkov-paul) [inline tools/test.py]:
```suggestion
                             'Usage: --update_config total_epochs=75 data.val.ann_file=my_annotation.json')
```

File: tools/test.py (hunk 1 above)
Comment (Ilya-Krylov) [reply tools/test.py:278]: ok, I see

**Comment by Ilya-Krylov** [comment]
run tests

**Review by druzhkov-paul — APPROVED** [review]

**Pull request merged**